test/browser/components/modals/save-map-modal.spec.js

```diff
@@ -1,5 +1,25 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Do we needs to add this? 
- If we do, can't we just add  a one liner `// Copyright (c) 2023 Uber Technologies, Inc., MIT license`? That is what we do in other vis.gl frameworks.
- Should this be a FSQ copyright? Or is the year wrong? I assume Uber didn't do anything with this code in 2023.

Reply:
this is part of the open source code. only files in the example folder should have the FSQ copyright